table-image-validation.suite/Scripts/Anne.script

```diff
@@ -0,0 +1,26 @@
+﻿Connect name:ResourcePath("screenshot.png"), type:"screenshot"
+
+put imagerectangle("anne/INNETWORK") into inNetworkColumn
+put ImageRectangle("anne/OUTOFNETWORK") into outNetworkColumn
+put imagelocation("anne/Box Top Corner") into topoftable
+
+put everyimagelocation(image:"anne/LineDivider", SearchRectangle:[topoftable.x, topoftable.y, (topoftable.x+imageinfo("anne/LineDivider").ImageSize.x), the remoteScreenSize.y])  into linelist //need to be very precise with the length of the search rectangle to be the exact width of the linedivider
+log linelist
+
+put 1 into index
+
+repeat the number of items in linelist - 1 times
```

Comment:
You might consider submitting a refactor for this to include a specific validation of the _In-Network Hearing Exam_. Otherwise, cool to see it pull content from all cells!